webui/react/src/pages/ExperimentDetails/TrialsComparisonModal.tsx

```diff
@@ -175,17 +175,18 @@ export const TrialsComparisonTable: React.FC<TableProps> = ({
   const latestMetrics = useMemo(
     () =>
       trialsDetails.reduce((metricValues, trial) => {
-        metricValues[trial.id] = Object.values(trial.summaryMetrics ?? {}).reduce(
-          (trialMetrics, curMetricType: Record<string, MetricSummary> | undefined) => {
-            for (const [metricName, metricSummary] of Object.entries(curMetricType ?? {})) {
-              trialMetrics[metricName] = metricSummary.last;
-            }
-            return trialMetrics;
-          },
-          {},
-        );
+        metricValues[trial.id] = Object.values<Record<string, MetricSummary> | null>(
```

Comment:
I think I might have originally written this, but it's pretty cumbersome.

Reply:
yeah, purposefully left it with minimal change for a couple of reasons:
1. I believe this area will be updated to support a variety of summary metrics in the near future not just `last`.
2. It's not going to be trivial to isolate an improvement in this area too much without impacting other areas, and also don't want to be blocking a high priority issue.